stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt

```diff
@@ -63,6 +63,8 @@ public class MessageMoreActionDialog(
                 dismiss()
             }
             binding.flagMessageButton.isVisible = style.flagMessageActionEnabled
+            binding.editMessageButton.isVisible = style.editMessageActionEnabled
+            binding.deleteMessageButton.isVisible = style.deleteMessageActionEnabled
```

Comment:
Why are we adding this to the branch where the message is not by the current user?

Reply:
Fixed